packages/frontend/src/components/DashboardFilter/ActiveFilters/ActiveFilter.tsx

```diff
@@ -71,36 +74,64 @@ const ActiveFilter: FC<Props> = ({
             closeOnEscape={!isSubPopoverOpen}
             closeOnClickOutside={!isSubPopoverOpen}
             onClose={handleClose}
+            offset={-1}
         >
             <Popover.Target>
                 <Tooltip
-                    withinPortal
                     position="top-start"
+                    disabled={isPopoverOpen}
                     label={
-                        filterRuleTables.length === 0
-                            ? `Table: ${filterRuleTables[0]}`
-                            : `Tables: ${filterRuleTables.join(', ')}`
+                        <Text fs="xs">
+                            {filterRuleTables.length === 0
+                                ? `Table: ${filterRuleTables[0]}`
+                                : `Tables: ${filterRuleTables.join(', ')}`}
+                        </Text>
                     }
                 >
-                    <div>
-                        <TagContainer
-                            interactive
-                            onRemove={onRemove}
-                            onClick={togglePopover}
-                        >
-                            {filterRule.label || filterRuleLabels.field}:{' '}
-                            {filterRule.disabled ? (
-                                <>is any value</>
-                            ) : (
-                                <>
-                                    {filterRuleLabels.operator}{' '}
-                                    <FilterValues>
-                                        {filterRuleLabels.value}
-                                    </FilterValues>
-                                </>
-                            )}
-                        </TagContainer>
-                    </div>
+                    <Button
+                        size="xs"
+                        variant={isTemporary ? 'outline' : 'default'}
+                        bg="white"
+                        rightIcon={
+                            (isEditMode || isTemporary) && (
+                                <ActionIcon
+                                    color="dark"
+                                    size="xs"
+                                    onClick={onRemove}
+                                >
+                                    <MantineIcon icon={IconX} />
+                                </ActionIcon>
+                            )
+                        }
+                        styles={{
+                            inner: {
+                                color: 'black',
+                            },
+                        }}
+                        onClick={togglePopover}
+                    >
+                        <Text fz="xs">
+                            <Text fw={600} span>
+                                {filterRule.label || filterRuleLabels.field}{' '}
```

Comment:
What do these white space strings do? Is it just for spacing?

Reply:
just for spacing!